src/redux/booking/sagas.js

```diff
@@ -7,7 +7,7 @@ export function* doSomething(action) {
 export function* watchForCreateBooking() {
   while (true) {
     yield take('CREATE_BOOKING_PENDING')
-
+    yield put({ type: 'CLEAR_ERRORS' })
```

Comment:
does this create a race condition between errors getting cleared and later (potentially) appended?